src/XMakeTasks/Microsoft.CSharp.targets

```diff
@@ -17,6 +17,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <!-- 
+        We are doing a cross-targeting traversal build if there is no list of target frameworks specified
+        nor is there a current target framework being built individually. In that case, this import is
+        redirected to Microsoft.Common.Traversal.targets.
+   -->
+   <PropertyGroup Condition="'$(TargetFrameworks)' != '' and '$(TargetFramework)' == ''">
```

Comment:
If there's only a single target framework specified, then the thing still loops, right? I remember some talk about single target frameworks not doing a loop. Did that change?